libs/sdk-react-native/android/src/main/java/com/breezsdk/BreezSDKModule.kt

```diff
@@ -1,27 +1,494 @@
+            promise.resolve(readableMapOf("status" to "ok"))
+        } catch (e: SdkException) {
+            e.printStackTrace()
+            promise.reject(TAG, "Error calling initServices", e)
+        }
+    }
+
+    @ReactMethod
+    fun start(promise: Promise) {
+        this.breezServices?.let {breezServices->
+            try {
+                breezServices.start()
+                promise.resolve(readableMapOf("status" to "ok"))
+            } catch (e: SdkException) {
+                e.printStackTrace()
+                promise.reject(TAG, "Error calling start", e)
+            }
+        } ?: run {
+            promise.reject(TAG, "BreezServices not initialized")
+        }
+    }
+
+    @ReactMethod
+    fun sync(promise: Promise) {
+        this.breezServices?.let {breezServices->
+            try {
+                breezServices.sync()
+                promise.resolve(readableMapOf("status" to "ok"))
+            } catch (e: SdkException) {
+                e.printStackTrace()
+                promise.reject(TAG, "Error calling sync", e)
+            }
+        } ?: run {
+            promise.reject(TAG, "BreezServices not initialized")
+        }
+    }
+
+    @ReactMethod
+    fun stop(promise: Promise) {
+        this.breezServices?.let {breezServices->
+            try {
+                breezServices.stop()
+                promise.resolve(readableMapOf("status" to "ok"))
+            } catch (e: SdkException) {
+                e.printStackTrace()
+                promise.reject(TAG, "Error calling stop", e)
+            }
+        } ?: run {
+            promise.reject(TAG, "BreezServices not initialized")
+        }
+    }
+
+    @ReactMethod
+    fun sendPayment(bolt11: String, amountSats: Double, promise: Promise) {
+        this.breezServices?.let {breezServices->
+            try {
+                var optionalAmountSats = amountSats.takeUnless { it == 0.0 }
+                var payment = breezServices.sendPayment(bolt11, optionalAmountSats?.toULong())
+
+                promise.resolve(readableMapOf(payment))
+            } catch (e: SdkException) {
+                e.printStackTrace()
+                promise.reject(TAG, "Error calling sendPayment", e)
+            }
+        } ?: run {
+            promise.reject(TAG, "BreezServices not initialized")
+        }
+    }
+
+    @ReactMethod
+    fun sendSpontaneousPayment(nodeId: String, amountSats: Double, promise: Promise) {
+        this.breezServices?.let {breezServices->
+            try {
+                var payment = breezServices.sendSpontaneousPayment(nodeId, amountSats.toULong())
+
+                promise.resolve(readableMapOf(payment))
+            } catch (e: SdkException) {
+                e.printStackTrace()
+                promise.reject(TAG, "Error calling sendSpontaneousPayment", e)
+            }
+        } ?: run {
+            promise.reject(TAG, "BreezServices not initialized")
+        }
```

Comment:
```suggestion
            try {
                var payment = getBreezServices().sendSpontaneousPayment(nodeId, amountSats.toULong())

                promise.resolve(readableMapOf(payment))
            } catch (e: SdkException) {
                e.printStackTrace()
                promise.reject(TAG, "Error calling sendSpontaneousPayment", e)
            }
        } 
```

Perhaps we can simplify a bit this patter that return on every method? It would need to create a method getBreezServices() that returns the initialized services or throws.

Reply:
I'll do that, for swift also